raw_description/urdf/vacuum_gripper/gripper.urdf.xacro

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:include filename="$(find raw_description)/urdf/vacuum_gripper/gripper.gazebo.xacro" />
+
+  <xacro:property name="gripper_length" value="0.14"/>
+  <xacro:property name="gripper_width" value="0.2"/>
+  <xacro:property name="gripper_height" value="0.14"/>
+  <xacro:property name="gripper_body_height" value="0.14"/>
+  <xacro:property name="gripper_radius" value="0.005"/>
+
+  <xacro:macro name="vacuum_gripper" params="name parent *origin">
+
+    <!-- joint between arm_ee_link and raw_boxgripper_link -->
+    <joint name="${name}_base_joint" type="fixed" >
+        <insert_block name="origin" />
+        <parent link="${parent}" />
+        <child link="${name}_frame" />
+    </joint>
+
+    <link name="${name}_frame"/>
+
+    <joint name="gripper_to_palm" type="fixed">
+        <parent link="${name}_frame"/>
+        <child link="${name}_body"/>
+        <origin xyz="0.08015 0.0475 0.0" rpy="1.57079632679 0 3.1415"/>
+    </joint>
+
+    <link name="${name}_body">
+        <visual>
+            <origin xyz="0 0 -0.048" rpy="0 0 0"/>
+            <geometry>
+                <mesh filename="package://raw_description/meshes/vacuum_gripper/vacuum_gripper.stl" />
+            </geometry>
+            <material name="silver">
+                <color rgba="0.8 0.8 0.8 1"/>
+            </material>
+        </visual>
+        <collision>
+            <origin xyz="0 0.0 0.0"/>
```

Comment:
Well, either you set it the **same** as the origin in visual (https://github.com/ipa320/cob_common/pull/207/files#diff-eb5b6ad15d50de8aa818f6122bd23fddR31)

or you modify the stl in blender so that you can remove **both** origins in visual **and**collision**